pkg/crmanager/validate.go

```diff
@@ -92,5 +92,12 @@ func (crMgr *CRManager) checkValidTransportServer(
 		return false
 	}
 
+	if len(tsResource.Spec.Type) == 0 {
```

Comment:
can we use tsResource.Spec.Type=="" as it is of type string.

Reply:
Done